Bottom: 1164998f042faa3be11dc5012f4df6855493af0d
Top:    07ecb2e517d696211efe3b15de2ce0da2bafa51b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-07 21:01:11 +0100

Refresh of hid-gtk-connect-up-the-file-ch

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 5a98a9f..e598d46 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -390,6 +390,7 @@ update_board_mtime_from_disk (void)
 
   ghidgui->our_mtime.tv_sec = 0;
   ghidgui->our_mtime.tv_usec = 0;
+  ghidgui->last_seen_mtime = ghidgui->our_mtime;
 
   if (PCB->Filename == NULL)
     return;
@@ -405,6 +406,8 @@ update_board_mtime_from_disk (void)
 
   g_file_info_get_modification_time (info, &ghidgui->our_mtime);
   g_object_unref (info);
+
+  ghidgui->last_seen_mtime = ghidgui->our_mtime;
 }
 
   /* Sync toggle states that were saved with the layout and notify the
